python/tvm/meta_schedule/utils.py

```diff
@@ -95,6 +95,17 @@ def get_global_func_with_default_on_worker(
         ) from error
 
 
+def get_global_func_on_rpc_session(session: RPCSession, name: str) -> PackedFunc:
+    try:
+        result = session.get_function(name)
+    except AttributeError as error:
+        raise AttributeError(
+            f'Unable to find function "{name}" on remote RPC server. '
+            "Please make sure USE_RANDOM is turned ON in the config.cmake on the RPC server."
```

Comment:
Is this must USE_RANDOM?

Reply:
Yeah I copy the error message from Ansor actually. Would you like to suggest changes?